webots_ros2_epuck/worlds/epuck_world.wbt

```diff
@@ -26,15 +26,13 @@ WoodenBox {
   name "wooden box 3"
   size 0.1 0.1 0.1
 }
-E-puck {
+E-puck_enu {
   controller "<extern>"
   version "2"
   camera_width 640
   camera_height 480
-  turretSlot [
-    Pi-puck {
-    }
-  ]
```

Comment:
I assume you removed the extension because the Pi-puck extension id not enu compatible?

Reply:
Yes. It is not a problem changing it, but we already have a lot of code in two places (e-pucks are in `webots` and `webots_ros2`). But if we agree to copy necessary models (converted to ENU) here I will return Pi-puck.

Reply:
Maybe we can fix this in a more general way just by opening the `translation` and `rotation` fields of the `Pi-puck` PROTO.

Reply:
I would prefer not opening these fields just for fixing a temporary problem. Rather I believe it's better to fix the Pi-puck PROTO on this PR (and in the upcoming FLU oriented e-puck).

Reply:
Honestly, opening a field looks less a hack than duplicating the PROTO to me.

Reply:
I don't like opening these fields, because that means there is a possibility to move the Pi-puck card on top the e-puck robot and it will still work, which is not true with the real robot. But we can implement it temporarily in this PR to fix the problem if you want. However, when we publish the FLU oriented e-puck, I would like to remove these fields.

Reply:
If it is temporarily, then I agree it is better to do correct since the begining and go for FLU.

Reply:
I've just checked and it works fine without conversion, textures and images were missing though.